packages/vscode-extension/src/webview/components/UrlBar.tsx

```diff
@@ -117,8 +130,9 @@ function UrlBar({ disabled }: { disabled?: boolean }) {
         }}
         recentItems={recentUrlList}
         items={sortedUrlList}
-        value={urlList[0]?.id}
-        disabled={disabledAlsoWhenStarting || urlList.length < 2}
+        key={urlSelectKey}
```

Comment:
setting this here is enough to re-mount the whole component. It doesn't have to be accessed in the underlying component